packages/@aws-cdk/cloudformation-include/test/test-templates/yaml/short-form-conditionals.yaml

```diff
@@ -0,0 +1,21 @@
+Conditions:
+  AlwaysTrueCond:
+    !Not
+    - !Equals
+      - Ref: AWS::Region
+      - completely-made-up-region
+  AndCond:
+    !And
+    - Condition: AlwaysTrueCond
+    - !Or
+      - Condition: AlwaysTrueCond
+      - Condition: AlwaysTrueCond
+Resources:
+  Bucket:
+    Type: AWS::S3::Bucket
+    Properties:
+      BucketName: 
+        !If
+        - AlwaysTrueCond
+        - MyBucketName
+        - Unreachable
```

Comment:
Again, this is not a legal CFN template. You can always use `!Ref AWS::NoValue` in these moments.

Reply:
Should I replace the "Unreachable" values in the JSON tests as well?

Reply:
Nah, just leave those for now.